livre/core/CMakeLists.txt

```diff
@@ -27,10 +27,6 @@ set(LIVRECORE_HEADERS
   configuration/Configuration.h
   configuration/Parameters.h
   data/DataSource.h
-  events/EventHandler.h
```

Comment:
I am not sure why those are removed. I want to keep equalizer functionality separated from the code. If you have a better suggestion we can surely apply, but I would like to keep them.